backend/app/infrastructure/__init__.py

```diff
@@ -56,20 +60,62 @@ def __getattribute__(self, name):
         return attr
 
 
+def lookup_max_time():
+    assert max_time is not None
+    return max_time
+
+
+def is_graph_error_permanent(error):
+    # TODO this must differentiate between different sources reasons for
+    #      failure
+    # Figure out where exactly is string about reorgs is hidden
+    try:
+        print(f"got TransportQueryError.query_id: {error.query_id}")
+        print(f"got TransportQueryError.errors: {error.errors}")
+        print(f"got TransportQueryError.data: {error.data}")
+        print(f"got TransportQueryError.extensions: {error.extensions}")
+    except:  # noqa: B001, E722
```

Comment:
use specific error that you expect here, it's only TransportQueryError for now, i guess; when another errors will be comming out, we'll be progressively adjusting the code

Reply:
+1

Reply:
Removed whole thing - it is not really needed.